ambry-router/src/test/java/com.github.ambry.router/MockServer.java

```diff
@@ -126,42 +230,50 @@ private void updateBlobMap(PutRequest putRequest)
   }
 
   /**
-   * Set the error for each request from this point onwards that affects subsequent PutRequests sent to this node
+   * Return the datacenter name of this server.
+   * @return the datacenter name.
+   */
+  String getDataCenter() {
+    return dataCenter;
```

Comment:
why should we? what difference does that make? Besides, that would be like a one-off thing.